CHANGELOG.md

```diff
@@ -5,11 +5,25 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.5.0] - 2015-12-22
+### Changed
+- updated the way makefiles are generated
+
+### Fixed
+- Fixed problem when tools use out of tree builds
```

Comment:
`out-of-tree` could stand hyphenation. This might need a little more detail, though.